### PR TITLE
RMMIS-5610-medicaid-entrylist-missing-header

### PR DESCRIPTION
**Pull request by kjschulz**
RMMIS-5610: Added Actions table header text on EntryList, misc cleanup and ES6ification

### CODE REVIEW

File: src/EntryList.jsx [hunk 1]
```diff
@@ -70,24 +70,22 @@ module.exports = React.createClass({
...
   },
 
-  componentDidMount: function() {
+  componentDidMount() {
     let propsId = this.props.id;
     // when the user clicks the show form button
     Dispatcher.register(`show-entrylist-form-${propsId}`, (action, data) => {
-      if( action === ENTRYLIST_FORM_SHOW &&
-           data.id === this.props.id) {
+      if( action === ENTRYLIST_FORM_SHOW && data.id === this.props.id) {
```
Comment (mallenscdhhs) [inline src/EntryList.jsx]: clean up `if` syntax: `if (action`

File: src/EntryList.jsx [hunk 2]
```diff
@@ -70,24 +70,22 @@ module.exports = React.createClass({
...
         this.setState({showForm: true});
       }
     });
 
     // when the user clicks the cancel button
     Dispatcher.register(`cancel-entrylist-entry-${propsId}`, (action, data) => {
-      if( action === ENTRYLIST_ENTRY_CANCEL &&
-           data.id === this.props.id) {
+      if( action === ENTRYLIST_ENTRY_CANCEL && data.id === this.props.id) {
```
Comment (mallenscdhhs) [inline src/EntryList.jsx]: clean up `if` syntax

File: src/EntryList.jsx [hunk 3]
```diff
@@ -170,17 +168,17 @@ module.exports = React.createClass({
...
 
-  showEmptyText: function() {
+  showEmptyText() {
     if (!this.state.entries.length) {
       return <tr><td colSpan={this.props.columns.length + 2}>{this.props.emptyText}</td></tr>;
     }
   },
 
-  getClassNames: function(){
+  getClassNames(){
```
Comment (mallenscdhhs) [inline src/EntryList.jsx]: need whitespace after `()`

File: src/EntryList.jsx [hunk 4]
```diff
@@ -200,8 +198,8 @@ module.exports = React.createClass({
               {columns.map(function(col) {
                 return <th key={col.dataKey}>{col.header}</th>;
               })}
-              <th key="edit"></th>
-              <th key="remove"></th>
+              <th key="actions" colSpan="2">Actions</th>
+
```
Comment (mallenscdhhs) [inline src/EntryList.jsx]: :no_good: not sure you need extra whitespace here

File: src/EntryList.jsx (hunk 1 above)
Comment (kjschulz) [inline src/EntryList.jsx]: Updated.

File: src/EntryList.jsx (hunk 2 above)
Comment (kjschulz) [inline src/EntryList.jsx]: Updated.

File: src/EntryList.jsx (hunk 3 above)
Comment (kjschulz) [inline src/EntryList.jsx]: Updated.

File: src/EntryList.jsx (hunk 4 above)
Comment (kjschulz) [inline src/EntryList.jsx]: !:no_good:  Updated.

**Comment by mallenscdhhs** [comment]
:+1: :8ball: :accept:

**Pull request merged**